### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -20948,6 +20961,7 @@ ubiquituously->ubiquitously
 ublisher->publisher
 ubunut->Ubuntu
 ubutunu->Ubuntu
+udno->undo
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest uno

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -18795,6 +18802,12 @@ solarmutx->solarmutex
 solatary->solitary
 soler->solver, solar, solely,
 soley->solely
+solf->solve
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest sold

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -18795,6 +18802,12 @@ solarmutx->solarmutex
...
+solfed->solved
+solfer->solver
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest solder

**Comment by luzpaz** [comment]
@peternewman requested revisions: f00cd3d

**Pull request merged**